sktime/utils/tests/test_ts_format.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf-8 -*-
+import os
+import tempfile
+from sktime.utils.ts_format import csv_to_ts_format
+
+
+def test_csv_to_ts_format():
+    # Testing a single dimension, no class label
+    fd, path = tempfile.mkstemp()
```

Comment:
`pytest` has some functionality for temp dirs which takes care of clean up automatically: https://docs.pytest.org/en/stable/tmpdir.html